tests/unit/moduleapi/moduleconfigs.tcl

```diff
@@ -243,5 +245,14 @@ start_server {tags {"modules"}} {
             assert_equal [r config get moduleconfigs.memory_numeric] "moduleconfigs.memory_numeric 1024"
         }
     }
-}
+    test {Module Update Args } {
+       r module load $testmoduleparameter 10 20 30
 
+       set t [r module list]
+       set modulename [lmap x [r module list] {dict get $x name}]
+       assert_not_equal [lsearch $modulename moduleparameter] -1
+       string match "10 20 30" [lmap x [r module list] {dict get $x args}]
+       r testmoduleparameter.update.parameter 40 50 60 70
+       string match "40 50 60 70" [lmap x [r module list] {dict get $x args}]
```

Comment:
This is not an assert. It will silently fail.

Reply:
Sorry, I use incorrect grammar, Now it is addressed by using assert_equal